app/code/Magento/CustomerGraphQl/etc/graphql/di.xml

```diff
@@ -6,6 +6,9 @@
  */
 -->
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
+    <type name="Magento\CustomerGraphQl\Model\Customer\CreateCustomerAccount">
```

Comment:
Hi @sedonik,
It's bad practice to create a plugin for class from the same module. Plugins are needed as a mechanism for extensibility from other modules.  
We can move this logic directly in `Magento\CustomerGraphQl\Model\Customer\CreateCustomerAccount` (or encapsulate check login in a separate class and use it resolver)

Thanks